manifests/init.pp

```diff
@@ -119,27 +119,39 @@
         }
         default:{
           case $version {
-            /^1\.8.*$/:{
+            /^1\.8(\..*)*$/:{
```

Comment:
As far as I know ruby devs are following semantic versioning which I think will always result in a version of the format $major.$minor.$patch. 

But I just realized debian package names either contain two or three digits (e.g.  ruby1.9.1, ruby2.1). Therefore your solution might be more logical for most users.